client/src/snippets.json

```diff
@@ -104,7 +104,7 @@
     "Boolean parameter": {
         "prefix": "gx-boolean-param",
         "body": [
-            "<param argument=\"${1:--}\" type=\"boolean\" truevalue=\"$1\" falsevalue=\"$2\" label=\"$3\" help=\"${4:TODO}\" />"
+            "<param argument=\"${1:--}\" type=\"boolean\" truevalue=\"$1\" falsevalue=\"$2\" checked=\"false\" label=\"$3\" help=\"${4:TODO}\" />"
```

Comment:
Is the intension to indicate that there is an checked option? false is the standard and adds just boilerplate. I guess it's fine to add, immjust wondering why :-)

Reply:
Just I think that including it can speed up the definition of booleans (in case the checked value needs to be set as "true".

Reply:
Your call. Removing it can be more work than adding it, if you need to remove it on average more.

Reply:
But if by default it is set as `checked="false"`, then it is only required to modify it when should be  `checked="true"`, doesn't it?

Reply:
Thanks for bringing up the discussion around default values! Maybe this https://github.com/galaxyproject/galaxy-language-server/issues/175 can help reduce the boilerplate while keeping the *flexibility* and *speed* when filling the snippets.

Reply:
```suggestion
            "<param argument=\"${1:--}\" type=\"boolean\" truevalue=\"$1\" falsevalue=\"$2\" checked=\"${3|false,true|}\" label=\"$4\" help=\"${5:TODO}\" />"
```
This gives the possibility to quickly change `checked` to `true` while navigating the placeholders with the TAB key.

@gallardoalba, if you are curious here is [a guide about the snippets syntax](https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax) that may be interesting for you.

Reply:
Yes, it seems very interesing. Thanks!